code/game/objects/items/lc13_skill_books.dm

```diff
@@ -0,0 +1,224 @@
+// LC13 SKILLS / ACTIONS
```

Comment:
Also a really bad itempath. 
Try items/fixerskills and then subdivide by level.